clusterloader2/pkg/prometheus/manifests/exporters/windows_node_exporter/windows-exporter-servicemonitor.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor 
+metadata:
+  labels:
+    k8s-app: windows-exporter
+  name: windows-exporter
+  namespace: monitoring
+spec:
+  endpoints:
+  - interval: 30s
+    port: http
+  jobLabel: k8s-app
+  namespaceSelector:
+    matchNames:
+    - kube-system
+  selector:
+    matchLabels:
+      k8s-app: windows-exporter
```

Comment:
Service monitor and PodMonitor aren't both needed.   This label won't match the Windows-exporter Deamon set above either. it would need to be `app: windows-exporter`  The namespace selector is off too.

I am not sure this projects perfered method (servicemonitor vs podmonitor)